openedx/core/djangoapps/content/search/documents.py

```diff
@@ -170,10 +170,10 @@ def _tags_for_content_object(object_id: UsageKey | LearningContextKey) -> dict:
     }
     for obj_tag in all_tags:
         # Add the taxonomy name:
-        if obj_tag.name not in result[Fields.tags_taxonomy]:
-            result[Fields.tags_taxonomy].append(obj_tag.name)
+        if obj_tag.taxonomy.name not in result[Fields.tags_taxonomy]:
```

Comment:
@bradenmacdonald On https://github.com/openedx/openedx-learning/pull/172 `_name` has been changed to `_export_id`, but here I have kept the taxonomy name. That is ok?

Reply:
Yes, that's right. Here should be the taxonomy name only. Thanks :)